sbi/inference/posterior.py

```diff
@@ -559,23 +641,6 @@ def _pyro_mcmc(
 
         return samples
 
-    # NOTE: this is done here because NeuralPosterior is created in inference methods
-    # at instantiation, while it could and should be created at training time.
-    def set_embedding_net(self, embedding_net: nn.Module) -> None:
-        """
-        Set the `embedding_net` as an attribute of the `neural_net`.
-
-        Args:
-            embedding_net: Neural net to encode `x`.
-        """
-        assert isinstance(embedding_net, nn.Module), (
-            "`embedding_net`is not a `nn.Module`. "
-            "If you want to use hard-coded summary features, "
-            "please simply pass the already encoded summary features as input and pass "
-            "`embedding_net=None`."
-        )
-        self.net._embedding_net = embedding_net
-
```

Comment:
Removing this since network creation is handled differently since https://github.com/mackelab/sbi/pull/256